collectors/modality-probe-udp-collector/src/opts.rs

```diff
@@ -44,7 +44,7 @@ impl From<Opts> for Config {
             output_file: o.output_file.unwrap_or_else(|| {
                 std::env::current_dir()
                     .expect("Could not retrieve current directory")
-                    .join(format!("session_{}_log_entries.csv", session_id))
+                    .join(format!("session_{}_log_entries.mtf", session_id))
```

Comment:
Multi-Track-File? If we're doing line-separated JSON, then the recommended practice is naming it `.jsonl`, I think. (Per: http://jsonlines.org/ )

Reply:
This is a @nathanaschbacher-ism. It's "modailty trace file". I'm happy to change it to whatever, though.